sdks/java/core/src/main/java/org/apache/beam/sdk/Pipeline.java

```diff
@@ -240,6 +242,35 @@ public void setCoderRegistry(CoderRegistry coderRegistry) {
      * visited.
      */
     public void visitValue(PValue value, TransformTreeNode producer);
+
+    /**
+     * Control enum for indicating whether or not a traversal should process the contents of
+     * a composite transform or not.
+     */
+    public enum Recurse {
```

Comment:
Calling this `Recurse` seems weird. Maybe `CompositeDisposition`? `CompositeBehavior`? `ShouldRecurse`? `RecursionBehavior`? ...